packages/@dcl/sdk-commands/src/commands/start/server/routes.ts

```diff
@@ -4,12 +4,23 @@ import { Router } from '@well-known-components/http-server'
 import { setupEcs6Endpoints } from './endpoints'
 import { PreviewComponents } from '../types'
 import { upgradeWebSocketResponse } from '@well-known-components/http-server/dist/ws'
+import { handleDataLayerWs } from '../data-layer/ws'
+import { RpcServer } from '@dcl/rpc'
+import { DataLayerContext } from '../data-layer/rpc'
 
-export async function wireRouter(components: PreviewComponents, dir: string) {
+export async function wireRouter(components: PreviewComponents, dir: string, rpcServer: RpcServer<DataLayerContext>) {
   const router = new Router<PreviewComponents>()
 
   const sceneUpdateClients = new Set<WebSocket>()
 
+  router.get('/data-layer', async (ctx, next) => {
```

Comment:
I'd only enable this using a `--data-layer` (or better arg name) to prevent exposing a filesystem API by default in servers like "sdk-test-scenes"